src/routes/RequestsRoute.js

```diff
@@ -66,6 +66,7 @@ import { getFormattedYears } from './utils';
 
 const INITIAL_RESULT_COUNT = 30;
 const RESULT_COUNT_INCREMENT = 30;
+const DISPLAYED_YEARS_AMOUNT = 3;
```

Comment:
@Stanislau-Snapok I see the same constant in `TitleInformation` perhaps it would be good to extract it to `constants` so it can be reused? Or perhaps the second argument in getFormattedYears can default to `DISPLAYED_YEARS_AMOUNT`.

Reply:
Were extracted into a global file with constants. Thanks!